packages/app/src/common.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
Not sure we need this. The homepage and learning path should be 2 different schemas.
ie homepage:
```
iconUrl: string;
label: string;
url: string;
```

and learning path
```
title: string;
url: string;
description?: string;
hours?: number;
paths?: number;
```

Reply:
Resolved in the latest commit.